Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -88,7 +88,8 @@ lemma enorm_eq_nnnorm (x : E) : ‖x‖ₑ = ‖x‖₊ := rfl
 @[simp, norm_cast] lemma enorm_lt_coe : ‖x‖ₑ < r ↔ ‖x‖₊ < r := by simp [enorm]
 
 @[simp] lemma enorm_ne_top : ‖x‖ₑ ≠ ∞ := by simp [enorm]
-@[simp] lemma enorm_lt_top : ‖x‖ₑ < ∞ := by simp [enorm]
+@[aesop (rule_sets := [finiteness]) safe apply, simp]
```

Comment:
The `≠` version should be tagged with `finiteness` instead:
* for all other lemmas the `≠` is tagged
* The fact that `Ne.lt_top` is a safe rule and `ne_top_of_lt` is an unsafe rule, `aesop` tries to use `≠` as its intermediate representation that things are finite (but can use the `<` as well, although reluctantly).

Feel free to add a sentence of that effect to the `Finiteness` module doc.

Reply:
Thanks; changed (and added two sentences to the module docs).